marketplacetests/marketplace_gaia_test.py

```diff
@@ -33,11 +33,11 @@ def install_in_app_payments_test_app(self):
 
         self.homescreen = Homescreen(self.marionette)
 
-        if not self.apps.is_app_installed(self.APP_NAME):
+        if not self.apps.is_app_installed(self.app_name):
 
             marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
-            marketplace.launch()
-            details_page = marketplace.navigate_to_app(self.APP_NAME)
+            home_page = marketplace.launch()
```

Comment:
That was an error. It is meant to be used afterwards, rather than `marketplace`.